satpy/tests/writer_tests/test_cf.py

```diff
@@ -759,14 +754,14 @@ def _gm_matches(gmapping, expected):
                                             'grid_mapping_name': 'geostationary',
                                             'semi_major_axis': a,
                                             'semi_minor_axis': b,
-                                            'sweep_axis': None,
-                                            'name': 'geos'})
+                                            # 'sweep_angle_axis': None,
```

Comment:
I think sweep angle axis is only guaranteed with pyproj 2.5+ since that version now bases the CF output on the WKT version of the CRS instead of the PROJ string. Ok with requiring pyproj 2.5+ for this writer?

Reply:
I'm fine with it, but it might be too recent. Could we just check that `sweep_angle_axis` is `y` if it is there ?